doc/nrf/ug_matter_overview_network_topologies.rst

```diff
@@ -1,32 +1,31 @@
 .. _ug_matter_overview_network_topologies:
 
-Matter network overview
-#######################
+Matter network topology and concepts
+####################################
 
 .. contents::
    :local:
    :depth: 2
 
-Matter network can be composed of Ethernet, Wi-Fi, and Thread devices.
-Networks are joined together, which means that devices that belong to different network can talk to each other within the local Matter fabric (infrastructure).
-All communication is achieved over IPv6, although the Matter network is capable of running in the absence of IPv6 infrastructure connected to the Internet.
+A Matter network can be composed of Ethernet, Wi-Fi, and Thread devices, which in turn can be part of one or more such IPv6-based networks.
```

Comment:
I don't quite understand the purpose of this sentence. Ethernet, Wi-Fi and Thread devices are always part of IPv6-based network, that usually are connected to another bigger IPv6 networks. I think that the previous version of this sentence was good enough.

Reply:
Brought back the first version of the sentence plus minor additions.